api/v1/middleware/auth.js

```diff
@@ -13,7 +14,11 @@ module.exports = function (req, res, next) {
 
 	return AuthService.verify(auth)
 		.then(function (decoded) {
-			req.auth = decoded;
+			req.auth = true;
```

Comment:
Might be a good idea to drop a comment there about it then, for future contributers